pkgs/development/python-modules/django-leaflet/default.nix

```diff
@@ -0,0 +1,32 @@
+{ pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, python3Packages
+}:
+
+buildPythonPackage rec {
+  pname = "django-leaflet";
+  version = "0.29.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-L23Ixxh/0i5itvK3tC7thpIPgb7DEqplSYHr5byOCGY=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    django
+  ];
+
+  # The tests seem to be impure.
+  # They are throwing a error about unset configs:
+  # > django.core.exceptions.ImproperlyConfigured: Requested setting LEAFLET_CONFIG, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
+  doCheck = false;
+
```

Comment:
Could we add `pythonImportsCheck` to have a least some sort of check?

Reply:
This sadly doesn't work either because of the same error doCheck throws. But the package in general seems to work because another package I'm currently working on uses it as a dependency and works fine.